<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartDesignGui::TaskChamferParameters</class>
 <widget class="QWidget" name="PartDesignGui::TaskChamferParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="message">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="buttonRefAdd">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;In toggle mode, you can select geometry either&lt;/p&gt;&lt;p&gt;in the list below, or in the 3D view to toggle&lt;/p&gt;&lt;p&gt;its state.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;In normal mode, you can only add new geometry&lt;/p&gt;&lt;p&gt;by selection. Existing geometry references can&lt;/p&gt;&lt;p&gt;be removed using the Delete key.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Toggle mode</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Remove per edge chamfer configuration</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidgetReferences">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string notr="true">Element</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Angle</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Flip</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="chamferType">
         <item>
          <property name="text">
           <string>Equal distance</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Two distances</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Distance and angle</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QToolButton" name="flipDirection">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Flip direction</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/PartDesign_Flip_Direction.svg</normaloff>:/icons/PartDesign_Flip_Direction.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="sizeLabel">
       <property name="text">
        <string>Default size</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Gui::QuantitySpinBox" name="chamferSize" native="true">
       <property name="keyboardTracking" stdset="0">
        <bool>false</bool>
       </property>
       <property name="value" stdset="0">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxUseAllEdges">
     <property name="text">
      <string>Use All Edges</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_1"/>
     <widget class="QWidget" name="page_2">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="size2Label">
         <property name="text">
          <string>Default Size 2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Gui::QuantitySpinBox" name="chamferSize2" native="true">
         <property name="keyboardTracking" stdset="0">
          <bool>false</bool>
         </property>
         <property name="value" stdset="0">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="angleLabel">
         <property name="text">
          <string>Default angle</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Gui::QuantitySpinBox" name="chamferAngle" native="true">
         <property name="keyboardTracking" stdset="0">
          <bool>false</bool>
         </property>
         <property name="minimum" stdset="0">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum" stdset="0">
          <double>180.000000000000000</double>
         </property>
         <property name="singleStep" stdset="0">
          <double>1.000000000000000</double>
         </property>
         <property name="value" stdset="0">
          <double>45.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>chamferType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
